router/cmd/main.go

```diff
@@ -82,5 +82,5 @@ func Main() {
 
 	profile.Finish()
 	logger.Debug("Server exiting")
-	os.Exit(0)
+	stop()
```

Comment:
💅 Maybe stop should be kept as a defer? Or is there some reason it can't? Avoid missing to call it for different paths where we return early?